web/client/src/util/apiWrapper.js

```diff
@@ -19,3 +19,28 @@ export const validateToken = (requestOptions) => {
     return JSON.parse(JSON.stringify({ Error: error }));
   });
 };
+
+export const getHTTPResponse = (values) => {
+  const param = {
+    url: values.URI,
+    method: values.httpMethod.toLowerCase(),
+    headers: {
+      Authorization: "Bearer " + values.token,
+      ...(values.contentType ? { "Content-Type": values.contentType } : {}),
+      ...(values.reqBody ? { data: values.reqBody } : {}),
+    },
+  };
+  let i;
+  for (i = 0; i < values.headers.length; i++) {
```

Comment:
Move the `let i` statement into the loop definition: `for (let i = 0; i < values.headers.length; i++) { ... }`

Reply:
done!